src/NLog.StructuredLogging.Json/INestedContext.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace NLog.StructuredLogging.Json
+{
+    public interface INestedContext : IDisposable
```

Comment:
I wonder if we can maybe make the interface and property names consistent with each other? The first two properties talk about scopes, but the type refers to nested?